otx/cli/tools/explain.py

```diff
@@ -44,16 +44,21 @@ def parse_args():
     """Parses command line arguments."""
 
     pre_parser = argparse.ArgumentParser(add_help=False)
-    pre_parser.add_argument("template")
-    parsed, _ = pre_parser.parse_known_args()
+    if os.path.exists("./template.yaml"):
```

Comment:
I think we need a unified function for finding template.yaml and that function could be placed to otx/cli/utils

Reply:
I've considered this method, but it looks like it needs some more thought. 
I'm not sure if it's appropriate to put a common function in `utils` for just lines 1-3 and then call the function inside the function. [main() -> parse_arg() -> get_template_arg..?]